core/lib/server/member.ts

```diff
@@ -68,17 +76,50 @@ export const getMembers = ({ communityId }: { communityId: CommunitiesId }, trx
 export const inviteMember = (props: NewMembers, trx = db) =>
 	autoRevalidate(
 		trx
+			.with("community_membership", (db) =>
+				db
+					.insertInto("community_memberships")
+					.values({
+						userId: props.userId,
+						communityId: props.communityId,
+						role: props.role ?? MemberRole.editor,
+					})
+					.returning("community_memberships.id")
+			)
 			.insertInto("members")
-			.values({
+			.values((eb) => ({
+				id: eb.selectFrom("community_membership").select("id") as unknown as MembersId,
```

Comment:
im assuming this is a stopgap solution for later right

Reply:
writing to both tables is a stopgap solution that is removed in my later PRs! but if you mean there's a better way to make the types work here, i would love to know!